src/imitation/scripts/train_rl.py

```diff
@@ -102,6 +105,12 @@ def train_rl(
         # so normalizing it makes training more stable. Note we do *not* normalize
         # observations here; use the `NormalizeFeaturesExtractor` instead.
         venv = VecNormalize(venv, norm_obs=False, **normalize_kwargs)
+        if reward_type == "RewardNet_normalized":
+            warnings.warn(
+                "Applying normalization to already normalized reward function. \
+                Consider removing the --normalize_reward flag",
```

Comment:
Seems like `normalize_reward` isn't necessarily a flag? It's an argument in the train_rl function. Especially for Sacred experiment we usually pass in `python -m imitation.scirpts.train_rl with normalize_reward=True`. 

Suggestion: `"Consider setting normalize_reward as False"` (especially the default for normalize_reward is True in imitaion.scripts.configs.train_rl)

Reply:
Oh yes, good catch.

Reply:
It is changed.